Makefile

```diff
@@ -181,9 +179,9 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 
 .PHONY: build-installer
 build-installer: manifests generate kustomize ## Generate a consolidated YAML with CRDs and deployment.
-	mkdir -p dist
+	mkdir -p release
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
-	$(KUSTOMIZE) build config/default > dist/install.yaml
+	$(KUSTOMIZE) build config/default > release/operator.yaml
```

Comment:
nit: happy to rename, but if not necessary, my experience is, changing from the operator-sdk templates makes it harder to update it later on for future operator-sdk versions. As long as it doesn't harm and we have no good reasons, we should not deviate from the defaults (even when we don't like it).

Reply:
Yeah I can revert this, updated because the naming `dist` did not make sense to me

Reply:
I see your point and leave it up to you. Not a big issue anyway. Maybe add a small comment that this deviates from the operator-sdk default for a reason, so that during a future alignment of the makefile people don't accidentally revert this change.